app/main/views/marketplace.py

```diff
@@ -271,29 +273,107 @@ def get_brief_by_id(framework_slug, brief_id):
     application_url = "/sellers/opportunities/{}/responses/create".format(brief['id'])
     add_case_study_url = None
 
+    profile_application_id = None
     profile_application_status = None
+    profile_application = None
+    supplier = None
+    unassessed_domains = {}
+    assessed_domains = {}
+    is_recruiter = False
+    product_seller = False
+    profile_url = None
+    recruiter_domain_list = []
+    supplier_assessments = {}
+    supplier_framework = None
+
     if current_user.is_authenticated:
+        if current_user.supplier_code is not None:
+            supplier = data_api_client.get_supplier(
+                current_user.supplier_code
+            ).get('supplier', None)
+
         profile_application_id = current_user.application_id
-        if profile_application_id is None:
-            profile_application_status = None
-        else:
+
+        if supplier is not None:
+            profile_url = '/supplier/{}'.format(supplier.get('code'))
+            assessed_domains = supplier.get('domains').get('assessed', None)
+            unassessed_domains = supplier.get('domains').get('unassessed', None)
+            legacy_domains = supplier.get('domains').get('legacy', None)
+
+            if profile_application_id is None:
+                profile_application_id = supplier.get('application_id')
+            is_recruiter = supplier.get('is_recruiter', False)
+            if is_recruiter == 'false':
+                is_recruiter = False
+            if is_recruiter == 'true':
+                is_recruiter = True
+            if is_recruiter:
+                for domain in supplier.get('recruiter_info').items():
+                    if int(domain[1].get('active_candidates')) > 0:
```

Comment:
the recruiter_info json is {'domain': {'active_candidates': '10', ...}}
so this creates a list of domains where the recruiter has more than 1 active candidate which is compared against the brief area of expertise to assess their ability to respond.